addons/website_forum/controllers/main.py

```diff
@@ -127,16 +133,22 @@ def questions(self, forum, tag=None, page=1, filters='all', my=None, sorting=Non
             'tag': str(tag.id) if tag else None,
             'filters': filters,
             'my': my,
+            'author_id': author.id if author else False,
```

Comment:
same, should be in the first commit (double check if other thing has been committing in the second commit instead of the first one)

Reply:
'author_id': author.id (will be false is no author anyway)
or even give author directly like below